system/modules/core/library/Contao/Folder.php

```diff
@@ -339,6 +340,7 @@ protected function getHash()
 	{
 		$arrFiles = array();
 
+		/** @var \RecursiveDirectoryIterator $it */
```

Comment:
Aah, got it. But using `@var \RecursiveDirectoryIterator[] $it` and a foreach-loop would be way more obvious ^^